source/basic.tex

```diff
@@ -3336,7 +3327,14 @@
 a subobject of such an object, and
 
 \item neither the original object nor the new object
-is a potentially-overlapping subobject\iref{intro.object}.
+is a potentially-overlapping subobject\iref{intro.object}, and
+
+\item neither the original object nor the new object are subobjects
+or both are subobjects, and
+
+\item the (unique) object (if any) that contains the original object
+as a subobject\iref{intro.object} is transparently replaceable by
+the (unique) object (if any) that contains the new object.
```

Comment:
The last should probably say "contains the new object as a subobject".

Reply:
Please consider #3796.